packages/eslint-plugin-ckeditor5-rules/lib/index.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
 		'no-legacy-imports': require( './rules/no-legacy-imports' ),
 		'no-relative-imports': require( './rules/no-relative-imports' ),
 		'ckeditor-error-message': require( './rules/ckeditor-error-message' ),
-		'ckeditor-imports': require( './rules/ckeditor-imports' ),
```

Comment:
This breaking change must be documented.

Reply:
> Or did you mean just adding a changeset entry with a Breaking Change specified?

Exactly. `ckeditor5-linters-config` is a standalone repository, and even if we do changes caused by CKEditor 5, we must document (describe) them in a release process.